### ISSUE TITLE
DropDown: Menu items incorrectly specified as role="option"

### ISSUE THREAD

**Issue by hross** [OfficeDev/office-ui-fabric-react#1449]
Per the wia aria specs, role='option' should only apply to listbox parents. In the case of Dropdown.tsx, the parent is a combobox. Per info here:

https://www.w3.org/TR/wai-aria/roles#option

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_listbox_role

https://www.w3.org/WAI/PF/aria/roles#combobox

We should be using the menu element.

**Comment by scriby** [comment]
Given that the Dropdown control doesn't allow text entry, should it not be using the listbox role instead of combobox? In that case it would make sense to stick with role=option on the items.

**Comment by scriby** [comment]
http://www.ssbbartgroup.com/blog/differences-aria-1-0-1-1-changes-rolecombobox/

Also some good background info there.

**Comment by scriby** [comment]
I was doing some testing in JAWS today and this change caused JAWS to not read the menu items, period.

Looking at the example on https://www.w3.org/TR/wai-aria/roles#combobox, the structure is to have the combobox element open a role=listbox element, and then use the role=option inside of the listbox.

**Comment by betrue-final-final** [comment]
That defeats the purpose of roles. Maybe @Chris<mailto:cschle@microsoft.com> has some insights.

________________________________
